hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.test.tsx

```diff
@@ -78,6 +85,29 @@ describe("<ProductOverview />", () => {
         expect(getByText("Add to cart")).toBeInTheDocument();
     });
 
+    test("Category label doesn't appear when there are no categories", async () => {
```

Comment:
<img width="497" alt="CleanShot 2022-12-23 at 16 23 51@2x" src="https://user-images.githubusercontent.com/36619969/209406704-8d1c029c-ee40-4e89-bee2-5df580e4ac78.png">

I believe if we import from Excel, this won't happen. But if we manually create the hardware item, it will not allow us to leave the category blank. We should change the database schema to make this field become optional.

Reply:
I'm not sure why this happens but the Django admin site puts a lot of fields as mandatory, even tho the database model says optional ,🤷🏻‍♀️ we can look into it more

Reply:
I will create a ticket on Jira to keep track of it @YuYing-Liang